eth_account/hdaccount/__init__.py

```diff
@@ -0,0 +1,30 @@
+from eth_utils import (
+    ValidationError,
+)
+
+from .deterministic import (
+    HDPath,
+)
+from .mnemonic import (
+    Mnemonic,
+)
+
+ETHEREUM_DEFAULT_PATH = "m/44'/60'/0'/0/0"
+
+
+def generate_mnemonic(num_words: int, lang: str) -> str:
+    return Mnemonic(lang).generate(num_words)
+
+
+def seed_from_mnemonic(words: str, passphrase: str) -> bytes:
```

Comment:
I'm a little confused by `passphrase` being a `str`.  Should we perform any validation at this stage such as ensuring that it is valid UTF8?

Reply:
`passphrase` is run through this validation (internally to `mnemonic`):
https://github.com/ethereum/eth-account/blob/d9868775981408ffad17ef66dc1fc72e127395fe/eth_account/hdaccount/_utils.py#L22

So, that should be enough to ensure it's a valid string?

Reply:
Here is where `normalize_string` is called:
https://github.com/ethereum/eth-account/blob/d9868775981408ffad17ef66dc1fc72e127395fe/eth_account/hdaccount/mnemonic.py#L201

from:
https://github.com/ethereum/eth-account/pull/87/files/d9868775981408ffad17ef66dc1fc72e127395fe#diff-3125cb8009b2655e3d284520d25953dcR26